platform-ajax/src/test/java/com/softicar/platform/ajax/event/AjaxDomEventWithExceptionTest.java

```diff
@@ -44,4 +46,18 @@ public void test() {
 		// assert text that was appended before the Exception occurred is displayed
 		assertEquals("Hello", getText(output));
 	}
+
+	@Test
+	public void testWithNonSofticarUserException() {
+
+		// click exception-throwing button
+		click(nonSofticarUserExceptionButton);
+		waitForServer();
+
+		// assert modal alert is displayed
+		var alert = findModalAlertOrFail();
+		assertFocused(DomModalAlertMarker.CLOSE_BUTTON);
+		assertEquals("An internal program error occurred.", getText(alert.getContent()));
```

Comment:
I think we should use `AjaxI18n.AN_INTERNAL_PROGRAM_ERROR_OCCURRED` instead of a literal string